README.md

```diff
@@ -711,16 +711,33 @@ meant to be able to change with it.
     easier understanding and reading of a test.
 
   * <a name="factories"></a>
-    Use [Factory Bot](https://github.com/thoughtbot/factory_bot) to create test
-    objects in integration tests. You should very rarely have to use
-    `ModelName.create` within an integration spec. Do **not** use fixtures as they
-    are not nearly as maintainable as factories.
+    Use [Factory Bot](https://github.com/thoughtbot/factory_bot) to
+    create test data in integration tests. You should very rarely
+    have to use `ModelName.create` within an integration spec. Do
```

Comment:
I believe the `ModelName.create` example is specific to Rails. Should we consider moving this section down to `Rails - models` section, or perhaps just add a note in the text?

Reply:
The wording implies Rails, but the advice itself is quite generic. Fixtures here are some objects kept in an storage, not even necessarily database. Also:

> Although factory_bot is written to work with ActiveRecord out of the box, it can also work with any Ruby class.

I'd rather keep the rule in this section, but changed `ModelName.create` into something that applies both to ActiveRecord and, say, file system.